MANIFEST.in

```diff
@@ -3,6 +3,7 @@ include requirements.txt
 include qiskit_nature/py.typed
 recursive-include qiskit_nature *.txt
 graft qiskit_nature/drivers/second_quantization/gaussiand/gauopen
+graft qiskit_nature/second_q/drivers/gaussiand/gauopen
```

Comment:
Should this part be separated out so it could be part of a bug fix (backported) release for updated wheels. As it stands I assume the G16 driver in the new second_q location does not correctly find the extension?